Data/VoxelMaterials_planetary.sbc

```diff
@@ -1,786 +1,838 @@
-<?xml version="1.0"?>
```

Comment:
Could you make sure to not do any white space changes? I guess the line endings is set different on your editor, than in the repo, that's why diff shows all lines changed. Also please update readme.md file to mention planet support added